### DISCUSSION TITLE
Do Pre-releases work for github?

### DISCUSSION THREAD

**Discussion by Satanarious** [ImranR98/Obtainium#1558] [Q&A]
Is there a reason why no matter what app I install, if I check the pre-release button, on update, it always says, 
`Download package ID does not match existing App ID - x`. 

<img src="https://github.com/ImranR98/Obtainium/assets/42642682/9fe3c07d-3cec-4693-8f1a-e92216b6ce96" width=40%>

Does it work for everybody else?

**Comment by DwainZwerg** [comment]
Yes it work
